### PR TITLE
I literally made this map in a day. Let's play it.

### PR DESCRIPTION
**Pull request by SkullyRoberts**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: optional name here
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

### CODE REVIEW

**Comment by SkullyRoberts** [comment]
Yea... it's actually functional now. I guess it's done until I get some feedback

**Comment by deathride58** [comment]
🤮

**Comment by SkullyRoberts** [comment]
Sooooooooo... you don't like it?
I mean, it's suppossed to be crap. But it's suppossed to be fun crap.

File: tgstation.dme [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "_maps\map_files\RobinStation\memestation.dmm"
```
Comment (deathride58) [inline tgstation.dme:17]: This isn't how you add new maps. This include breaks the map loader

**Comment by Ghommie** [comment]
> 
> 
> Yea... it's actually functional now. I guess it's done until I get some feedback

Too many tiny and big (intended or not) wrong bits to list.
On the bonus side you gave me a daily dose of chuckle, thanks man.

File: tgstation.dme (hunk 1 above)
Comment (SkullyRoberts) [reply tgstation.dme:17]: Okay. I've never actually added a map before. How do I do it?

**Comment by SkullyRoberts** [comment]
Someday I will be able to compile SS13 on my laptop in less than an hour... that will be a good day...

**Comment by SkullyRoberts** [comment]
OH MY GOD IT ACTUALLY WORKED!

**Pull request closed without merging**